# empty request
---
onrobot_rg_driver/RGStatus status